<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="wsc-042" name="Test" tests="18" failures="6" timestamp="23 Jan 2017 12:56:59 GMT" time="0.078" errors="0">
  <testcase name="checkDivisionNegative" time="0.007" classname="com.CalculatorTest.DivisionTest"/>
  <testcase name="checkDivisionPositive" time="0.001" classname="com.CalculatorTest.DivisionTest"/>
  <testcase name="checkDivisionPositiveNegative" time="0.001" classname="com.CalculatorTest.DivisionTest"/>
  <testcase name="checkDivisionSelf" time="0.004" classname="com.CalculatorTest.DivisionTest">
    <failure type="java.lang.AssertionError" message="10/10 operation result is wrong! expected [1.0] but found [10.0]">
      <![CDATA[java.lang.AssertionError: 10/10 operation result is wrong! expected [1.0] but found [10.0]
	at com.CalculatorTest.DivisionTest.checkDivisionSelf(DivisionTest.java:45)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkDivisionSelf -->
  <testcase name="checkMultiplicationFload" time="0.001" classname="com.CalculatorTest.MultiplicationTest"/>
  <testcase name="checkMultiplicationNegative" time="0.001" classname="com.CalculatorTest.MultiplicationTest"/>
  <testcase name="checkMultiplicationOne" time="0.001" classname="com.CalculatorTest.MultiplicationTest">
    <failure type="java.lang.AssertionError" message="10*1 operation result is wrong! expected [10.0] but found [1.0]">
      <![CDATA[java.lang.AssertionError: 10*1 operation result is wrong! expected [10.0] but found [1.0]
	at com.CalculatorTest.MultiplicationTest.checkMultiplicationOne(MultiplicationTest.java:44)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkMultiplicationOne -->
  <testcase name="checkMultiplicationPositive" time="0.001" classname="com.CalculatorTest.MultiplicationTest"/>
  <testcase name="checkMultiplicationZero" time="0.001" classname="com.CalculatorTest.MultiplicationTest"/>
  <testcase name="checkSimpleSubtraction" time="0.001" classname="com.CalculatorTest.SubtractionTest">
    <failure type="java.lang.AssertionError" message="2 - 3 operation result is wrong! expected [-1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: 2 - 3 operation result is wrong! expected [-1.0] but found [0.0]
	at com.CalculatorTest.SubtractionTest.checkSimpleSubtraction(SubtractionTest.java:44)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkSimpleSubtraction -->
  <testcase name="checkSubtractionFloats" time="0.001" classname="com.CalculatorTest.SubtractionTest"/>
  <testcase name="checkSubtractionNegative" time="0.001" classname="com.CalculatorTest.SubtractionTest"/>
  <testcase name="checkSubtractionSelf" time="0.001" classname="com.CalculatorTest.SubtractionTest">
    <failure type="java.lang.AssertionError" message="1.1 - 1.1 operation result is wrong! expected [0.0] but found [1.1]">
      <![CDATA[java.lang.AssertionError: 1.1 - 1.1 operation result is wrong! expected [0.0] but found [1.1]
	at com.CalculatorTest.SubtractionTest.checkSubtractionSelf(SubtractionTest.java:57)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkSubtractionSelf -->
  <testcase name="checkZeroSubtraction" time="0.001" classname="com.CalculatorTest.SubtractionTest">
    <failure type="java.lang.AssertionError" message="3 - 0 operation result is wrong! expected [3.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: 3 - 0 operation result is wrong! expected [3.0] but found [0.0]
	at com.CalculatorTest.SubtractionTest.checkZeroSubtraction(SubtractionTest.java:30)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkZeroSubtraction -->
  <testcase name="checkAdditionFloats" time="0.001" classname="com.CalculatorTest.AdditionTest"/>
  <testcase name="checkAdditionNegative" time="0.001" classname="com.CalculatorTest.AdditionTest"/>
  <testcase name="checkSimpleAddition" time="0.001" classname="com.CalculatorTest.AdditionTest">
    <failure type="java.lang.AssertionError" message="2 + 3 operation result is wrong! expected [5.0] but found [2.0]">
      <![CDATA[java.lang.AssertionError: 2 + 3 operation result is wrong! expected [5.0] but found [2.0]
	at com.CalculatorTest.AdditionTest.checkSimpleAddition(AdditionTest.java:44)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkSimpleAddition -->
  <testcase name="checkZeroAddition" time="0.0" classname="com.CalculatorTest.AdditionTest"/>
</testsuite> <!-- Test -->
